src/Asset/AssetDefinition.php

```diff
@@ -991,6 +991,12 @@ private function validateCapacityArray(mixed $capacities, bool $check_values = t
      */
     private function getDecodedTranslationsField(): array
     {
+        // prevent warning when no translation are available (create step)
+        if (empty($this->fields['translations'])) {
+            $this->fields['translations'] = '[]'; // prevent warning to be triggered on each method call
+            return [];
+        }
```

Comment:
This may hide unexpected empty value on existing items.
Another possible fix is to check if item is new in the `getTranslatedName()` method:
```php
    public function getTranslatedName(int $count = 1): string
    {
        if ($this->isNewItem()) {
            return '';
        }
```

By the way, I do not even see where the translated name would be displayed in the creation form.

Reply:
:+1: